### PR TITLE
float broccoli-persistent-filter

### PR DESCRIPTION
**Pull request by kellyselden**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "dependencies": {
     "broccoli-merge-trees": "^1.1.1",
-    "broccoli-persistent-filter": "1.2.0",
+    "broccoli-persistent-filter": "^1.0.0",
```
Comment (Turbo87) [inline package.json]: shouldn't this be at least `^1.2.0`?

Comment (kellyselden) [inline package.json]: I don't think it matters. I don't see how someone could install something less than 1.2.0.

Comment (Turbo87) [inline package.json]: could happen when installed with `npm@3` and some other plugin using `"broccoli-persistent-filter": "1.0.0"`

Comment (kellyselden) [inline package.json]: Ah good point.

**Pull request merged**